services/proxy/src/Proxy/API.hs

```diff
@@ -68,6 +68,10 @@ sitemap e = do
         (proxy e "key" "secrets.googlemaps" Static "/maps/api/staticmap" googleMaps)
         return
 
+    get "/proxy/googlemaps/maps/api/geocode/:path"
+        (proxy e "key" "secrets.googlemaps" Prefix "/maps/api/geocode" googleMaps)
```

Comment:
Change to either

```haskell
    get "/proxy/googlemaps/maps/:path"
        (proxy e "key" "secrets.googlemaps" Prefix "/maps" googleMaps)
```
(using `Prefix`)

or 

```haskell
    get "/proxy/googlemaps/maps/api/geocode/:path"
        (proxy e "key" "secrets.googlemaps" Static "/maps/api/geocode" googleMaps)
```
(using `Static`)

Alternatively, ask clients to call `[...]/proxy/googlemaps/api/geocode/json?latlng=0,0` without the `/maps/` segment (which is consistent with the inconsistent existing googlemaps/api/static endpoint but inconsistent with regards to the other endpoints) and combine the two `/proxy/googlemaps/`.